src/core/Navigation/SideNavigation/SideNavigation/SideNavigation.tsx

```diff
@@ -14,29 +14,29 @@ import { getConditionalAriaProp } from '../../../../utils/aria';
 import { IconChevronDown, IconChevronUp } from 'suomifi-icons';
 
 export interface SideNavigationProps {
-  /** Use the `<SideNavigationItem>` components as children */
+  /** Use `<SideNavigationItem>` components as children */
   children: ReactNode;
-  /** Name for the navigation element. Don't use the word "navigation" since it will be read by screen reader regardless. */
+  /** Name for the navigation element. Don't use the word "navigation" since it will be read by screen readers regardless */
   'aria-label': string;
-  /** ID for the HTML nav element */
+  /** HTML id attribute for the `<nav>` element. Typically used together with a `<SkipLink>` to quickly move to the nav */
   id?: string;
   /**
-   * Normal or small screen variant
+   * Variant of the component
```

Comment:
"Variant of the component" sounds a bit off, maybe I'd replace these with "Component variant" instead

Reply:
This is now addressed in the latest commit